### PR TITLE
PLATOPS-2035: Removed playReactiveMongo instances and updated with Si…

### PR DESCRIPTION
**Pull request by swathimadhuri**
…mpleReactiveMongo in create.py file

### CODE REVIEW

File: scripts/bin/create.py [hunk 1]
```diff
@@ -141,8 +141,7 @@ def replace_variables_for_app(application_root_name, folder_to_search, applicati
     bootstrapPlay25Version=get_latest_library_version_in_open("bootstrap-play-25")
     govukTemplateVersion=get_latest_library_version_in_open("govuk-template")
     playUiVersion=get_latest_library_version_in_open("play-ui")
-    playReactivemongoVersion="6.2.0"
-    simpleReactivemongoVersion="6.1.0"
+    simpleReactivemongoVersion="7.15.1"
```
Comment (tomasz-rosiek) [inline scripts/bin/create.py]: Use 7.16.0 instead

Comment (konradwudkowski) [reply scripts/bin/create.py]: or get latest?

**Comment by dabd** [comment]
👍

File: templates/service/project/AppDependencies.scala [hunk 2]
```diff
@@ -12,7 +12,7 @@ object AppDependencies {
     "uk.gov.hmrc"             %% "play-ui"                  % "$!playUiVersion!$",
     <!--(end)-->
     <!--(if MONGO)-->
-    "uk.gov.hmrc"             %% "play-reactivemongo"       % "$!playReactivemongoVersion!$",
+    "uk.gov.hmrc"             %% "simple-reactivemongo"       % "$!simpleReactivemongoVersion!$",
```
Comment (konradwudkowski) [inline templates/service/project/AppDependencies.scala]: percent signs are no longer vertically aligned

Comment (dabd) [reply templates/service/project/AppDependencies.scala]: Fixed in https://github.com/hmrc/init-service/commit/547e9cdd38344cccbb1e904429264575a1a86eca

Comment (konradwudkowski) [reply templates/service/project/AppDependencies.scala]: thanks!

**Pull request merged**